### ISSUE TITLE
YAHE (Yet Another Highlighting Error)

### ISSUE THREAD

**Issue by solson** [nixeagle/ooc-mode#14]
```
// check for an alphanumeric character
isAlphaNumeric: func -> Bool {
    isAlpha() || isDigit()
}

// check for an alphabetic character
isAlpha: func -> Bool {
    isLower() || isUpper()
}
```

In these functions, `isDigit` and `isUpper` (the right sides of `||`) are highlighted as variable definitions. They shouldn't have any special highlighting, just the default colour like the left side of `||`.

**Comment by nixeagle** [comment]
This triggers after something like:
    || func()

This happens because the inline lambda arg stuff `|arg|` seems to be a little too greedy.

**Comment by nixeagle** [comment]
Ensure arguments exist in inline lambda list before triggering highlight.

Closed by 8a9b5649ab43e7179bf7a353879dd7b39524bb72
